From patchwork Thu Nov  7 02:27:06 2019
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: Peng Hao <richard.peng@oppo.com>
X-Patchwork-Id: 11231665
Return-Path: <SRS0=VFBJ=Y7=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 1C58B1709
	for <patchwork-kvm@patchwork.kernel.org>;
 Thu,  7 Nov 2019 02:36:17 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.kernel.org (Postfix) with ESMTP id 01E452075C
	for <patchwork-kvm@patchwork.kernel.org>;
 Thu,  7 Nov 2019 02:36:17 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1732790AbfKGCgO (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Wed, 6 Nov 2019 21:36:14 -0500
Received: from m17618.mail.qiye.163.com ([59.111.176.18]:51571 "EHLO
        m17618.mail.qiye.163.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S1727778AbfKGCgO (ORCPT <rfc822;kvm@vger.kernel.org>);
        Wed, 6 Nov 2019 21:36:14 -0500
X-Greylist: delayed 549 seconds by postgrey-1.27 at vger.kernel.org;
 Wed, 06 Nov 2019 21:36:12 EST
Received: from P80254710 (unknown [121.12.147.249])
        by m17618.mail.qiye.163.com (Hmail) with ESMTPA id 90E004E17EE;
        Thu,  7 Nov 2019 10:27:01 +0800 (CST)
Date: Thu, 7 Nov 2019 10:27:06 +0800
From: "Peng Hao" <richard.peng@oppo.com>
To: pbonzini <pbonzini@redhat.com>, rkrcmar <rkrcmar@redhat.com>
Cc: kvm <kvm@vger.kernel.org>,
        linux-kernel <linux-kernel@vger.kernel.org>
Subject: [PATCH] kvm/x86 : Replace BUG_ON(1) with BUG()
X-Priority: 3
X-GUID: 608D3069-12C2-46ED-919C-21192C18CDB7
X-Has-Attach: no
X-Mailer: Foxmail 7.2.10.151[en]
Mime-Version: 1.0
Message-ID: <201911071027048908705@oppo.com>
X-HM-Spam-Status: e1kfGhgUHx5ZQUtXWQgYFAkeWUFZVk9VTE5OS0tKTk9PQk5JSEJZV1koWU
        FJSUtLSjdXWS1ZQUlXWQkOFx4IWUFZNTQpNjo3JCkuNz5ZBg++
X-HM-Sender-Digest: e1kMHhlZQR0aFwgeV1kSHx4VD1lBWUc6Mwg6ITo6CDg6DBgMSlYYLy9M
        SjZPCTJVSlVKTkxIS0JITUlJSU5KVTMWGhIXVQkSGBMaCR9VCx4VHDsUCwsUVRgUFkVZV1kSC1lB
        WUpJSlVKSVVKT0xVSU9CWVdZCAFZQUpOQ0M3Bg++
X-HM-Tid: 0a6e43afc0f59376kuws90e004e17ee
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Signed-off-by: Peng Hao <richard.peng@oppo.com>
---
 arch/x86/kvm/vmx/nested.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

-- 
2.7.4

diff --git a/arch/x86/kvm/vmx/nested.c b/arch/x86/kvm/vmx/nested.c
index e76eb4f..d0e6c40 100644
--- a/arch/x86/kvm/vmx/nested.c
+++ b/arch/x86/kvm/vmx/nested.c
@@ -4945,8 +4945,7 @@ static int handle_invept(struct kvm_vcpu *vcpu)
 	 */
 		break;
 	default:
-		BUG_ON(1);
-		break;
+		BUG();
 	}
 
 	return nested_vmx_succeed(vcpu);
